### PR TITLE
Readmefixing

### PR DESCRIPTION
**Pull request by semuelle**
Some improvements to the readme:

- some phrasing
- some structure
- some links fixed
- highlighting details that are often missed, such as submitting a single file
- table of contents
- pretty icons for links

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,125 +24,122 @@
...
 
-**W3F Grants Evaluators**
+#### W3F Grants Evaluators
 
 Evaluators are individuals able to evaluate the technology delivered as a result of the Open Grants Program. The committee decides about the evaluators and has the right to add or remove evaluators (supermajority needed).
 
-* [Marcin Górny](https://github.com/mmagician)
-* [David Hawig](https://github.com/Noc2)
+- [Marcin Górny](https://github.com/mmagician)
+- [David Hawig](https://github.com/Noc2)
```
Comment (mmagician) [inline README.md:68]: Probably needs an expansion now :)

**Review by mmagician — APPROVED** [review]
Looks great! Thanks for the updates

File: README.md (hunk 1 above)
Comment (Noc2) [reply README.md:68]: Agree. I will take care of it.

**Comment by Noc2** [comment]
Looks really cool. Thanks. Maybe also update the general grants readme.

**Pull request merged**